app/assets/javascripts/components/students_card.es6.jsx

```diff
@@ -0,0 +1,59 @@
+class StudentsCard extends React.Component {
```

Comment:
Remember to change `React.Component` to just `Component` if you use the console generator!